vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-client/src/main/java/com/vaadin/client/ApplicationConnection.java

```diff
@@ -760,7 +582,7 @@ private boolean isExecutingDeferredCommands() {
      * @return The loading indicator for this ApplicationConnection
      */
     public VLoadingIndicator getLoadingIndicator() {
```

Comment:
It seems that both methods can be removed. Couldn't find them being used anywhere.

Reply:
<img width="1262" alt="Screenshot 2022-06-07 at 15 39 35" src="https://user-images.githubusercontent.com/1222264/172381478-23d9ce7c-1ef4-4297-9469-acc0161e35f3.png">